homeassistant/components/zamg/config_flow.py

```diff
@@ -72,14 +72,13 @@ async def async_step_user(
             )
 
         return self.async_create_entry(
-            title=user_input.get(CONF_NAME) or self._client.get_station_name,
+            title=self._client.get_station_name,
             data={CONF_STATION_ID: station_id},
         )
 
     async def async_step_import(self, config: dict[str, Any]) -> FlowResult:
         """Handle ZAMG configuration import."""
         station_id = str(config.get(CONF_STATION_ID))
```

Comment:
The station id is already a string coming from configuration.yaml.

Reply:
It is optional though so we need to handle that.

Reply:
you mean line 81?
that's why i use there `config.get()` instead of `config[CONF_STATION_ID]` and catch a not existing one below

Reply:
Converting `None` to `"None"` isn't intended I think.